include/random.h

```diff
@@ -163,6 +163,10 @@ enum RandomTag
     RNG_AI_ABILITY,
     RNG_AI_SWITCH_HASBADODDS,
     RNG_AI_SWITCH_WONDER_GUARD,
+    RNG_AI_SWITCH_TOXICD,
```

Comment:
I assume this is "Toxic'd" or something, which means I think it needs a better name. "Toxic" simply refers to the move, everything else refers to it as "bad poison" or "... was badly poisoned".